admin-dev/themes/new-theme/js/app/pages/stock/store/index.ts

```diff
@@ -22,13 +22,15 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
-import Vue from 'vue';
+/* eslint-disable no-param-reassign */
+
+import Vue, {PluginObject} from 'vue';
 import Vuex from 'vuex';
 import _ from 'lodash';
 import * as actions from './actions';
 import mutations from './mutations';
 
-Vue.use(Vuex);
+Vue.use(<PluginObject<any>><unknown>Vuex);
```

Comment:
Could you add a little comment, about why you have to cast it in `unknown`  then ` <PluginObject<any>>`, it's not really obvious ^^

Reply:
Needed to update Vuex in order to have the correct definition file, it's no more needed :)